server.js

```diff
@@ -4,12 +4,49 @@ const path = require('path');
 
 const app = express();
 
-// view engine setup
 app.set('views', path.join(__dirname, 'views'));
 app.set('view engine', 'hbs');
 app.use(express.static(path.join(__dirname, 'public')));
 
-// add your routes here :)
+app.get('/:code', (req, res) => {
+  const code = req.params.code;
+  const promise1 = fetch('http://localhost:3000/code/' + code)
```

Comment:
Not sure if you guys have dug into string templating at all, but you can use that here.

```js
const noStringTemplating = 'http://localhost:3000/code/' + code;

const withStringTemplating = `http://localhost:3000/code/${code}`;
```